sdk/src/main/java/co/omise/android/models/SourceType.kt

```diff
@@ -29,13 +29,22 @@ sealed class SourceType(
     object Dana : SourceType("dana")
     object Gcash : SourceType("gcash")
     object Kakaopay : SourceType("kakaopay")
-    object TouchNGo : SourceType("touch_n_go")
+    object TouchNGo : SourceType("touch_n_go") {
+        var provider : String? = null
+    }
     object Fpx : SourceType("fpx") {
         var banks : List<Bank>? = null
     }
     object RabbitLinePay : SourceType("rabbit_linepay")
     object OcbcPao : SourceType("mobile_banking_ocbc_pao")
-    object GrabPay : SourceType("grabpay")
+    object Boost: SourceType("boost")
+    object ShopeePay: SourceType("shopeepay")
+    object DuitNowOBW  : SourceType("duitnow_obw")
+    object DuitNowQR : SourceType("duitnow_qr")
+    object MaybankQR : SourceType("maybank_qr")
+    object GrabPay : SourceType("grabpay"){
+        var provider : String? = null
+    }
```

Comment:
Could you change to `data class` please?

Reply:
Updated here krub -> https://github.com/omise/omise-android/pull/225/commits/4fd9b21527ab952e5ec38956a94eb71c1109118b  🙏